.gitignore

```diff
@@ -198,4 +198,7 @@ UpgradeLog*.htm
 # Business Intelligence projects
 *.rdl.data
 *.bim.layout
-*.bim_*.settings
\ No newline at end of file
+*.bim_*.settings
+
```

Comment:
Not sure what you mean here, it looks OK to me. No new line at the end of the line.